org.darktable.Darktable.json

```diff
@@ -0,0 +1,124 @@
+{
+    "app-id": "org.darktable.Darktable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "darktable",
+    "rename-desktop-file": "darktable.desktop",
+    "rename-appdata-file": "darktable.appdata.xml",
+    "rename-icon": "darktable",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.secrets",
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2 -std=c++11",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/bin/exiv2*",
+        "/bin/*lensfun*",
+        "/bin/xml*",
+        "/bin/xslt*",
+        "/bin/metacopy",
+        "/bin/pathtest",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/man",
+        "*.a",
+        "*.la"
+    ],
+    "modules": [
+        {
+            "name": "lensfun",
+            "cmake": true,
+            "config-opts": [
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/lensfun/0.3.2/lensfun-0.3.2.tar.gz",
+                    "sha256": "ae8bcad46614ca47f5bda65b00af4a257a9564a61725df9c74cb260da544d331"
+                }
+            ]
+        },
+        {
+            "name": "libxml2",
+            "config-opts": [
+                "--with-python=no"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.gnome.org/libxml2"
+                }
+            ]
+        },
+        {
+            "name": "libxslt",
```

Comment:
Yes indeed! But there must be a reason, why I added those in the first place some months ago …